src/main/java/nextstep/courses/domain/session/SessionStatusEnum.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.courses.domain.session;
+
+public enum SessionStatusEnum {
+    PENDING,
+    OPEN,
+    CLOSED;
+
+    public boolean isSessionOpened() {
+        if (this != OPEN) {
```

Comment:
다음과 같이 하나의 라인으로 작성해볼 수 있을 것 같습니다.
```suggestion
        return this == OPEN;
```